### ISSUE TITLE
xenial: Unable to locate package gnome-session-fallback

### ISSUE THREAD

**Issue by jsnow0187** [dnschneid/crouton#2723]
```
Please paste the output of the following command here: sudo edit-chroot -all
```
uilding dependency tree       
Reading state information... Done
E: Unable to locate package #gnome-session-fallback
Failed to complete chroot setup.
The chroot setup script may be broken. Your chroot is not fully configured.
Removing the chroot setup script. You may want to update your chroot again.
UID 1000 not found in xenial
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...
#### Please describe your issue:

Anytime I have tried to install xenial, or anything other than trusty, I get some errors.  I posted the output above with xenial -gnome because thats what I would prefer to use, but I could not install vivid with xfce, xenial with unity, wily with lxde, etc....  It all comes down to some kind of missing package most of the time and the UID 1000 not found.  I was able to get xenial -gnome to install by editing prepare.sh and removing gnome-session-fallback and replacing it with gnome-session-flashback.  
#### If known, describe the steps to reproduce the issue:
just installing crouton with a distro other than trusty

**Comment by jsnow0187** [comment]
I forgot to add the output of edit-chroot -all

ame: trusty
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/trusty...
crouton: version 1-20160715134041~master:0dc76a22
release: trusty
architecture: amd64
xmethod: xorg
targets: gnome
host: version 8350.68.0 (Official Build) stable-channel winky 
kernel: Linux localhost 3.10.18 #1 SMP Tue Aug 2 21:10:21 PDT 2016 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/trusty...
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20160715134041~master:0dc76a22
release: xenial
architecture: amd64
xmethod: xorg
targets: gnome
host: version 8350.68.0 (Official Build) stable-channel winky 
kernel: Linux localhost 3.10.18 #1 SMP Tue Aug 2 21:10:21 PDT 2016 x86_64 x86_64 x86_64 GNU/Linux
freon: yes
Not unmounting /mnt/stateful_partition/crouton/chroots/xenial as another instance is using it.

**Comment by RMcNeely** [comment]
I had the same issues everytime I tried to target Gnome as the DE. I did get things working when I changed to installing XFCE though. It does seem to be failing b/c they renamed that packaged to `gnome-flashback` or something like that. I think if we found the line that calls that package and changed it, that it would solve the issue. 

`chronos@localhost ~/Downloads $ sudo edit-chroot -l xenial
name: xenial
encrypted: no
Entering /mnt/stateful_partition/crouton/chroots/xenial...
crouton: version 1-20160823173305~master:1e0c0488
release: xenial
architecture: armhf
xmethod: xiwi
targets: chromium,x11,xiwi,xfce,audio,core,gtk-extra,xorg,cli-extra
host: version 8737.1.0 (Official Build) dev-channel daisy 
kernel: Linux localhost 3.8.11 #1 SMP Thu Aug 25 00:43:54 PDT 2016 armv7l armv7l armv7l GNU/Linux
freon: yes
Unmounting /mnt/stateful_partition/crouton/chroots/xenial...`

**Comment by jsnow0187** [comment]
I found the line, i am able to get it installed just have to wait for it to fail and edit prepare.sh.  its a pain, i dont know how to edit it before the fact... Otherwise id fix it

**Comment by RMcNeely** [comment]
Can you link to the line?

**Comment by DennisLfromGA** [comment]
@RMcNeely,

The `prepare.sh` script is built on-the-fly depending on which targets you specify so the line number in question will be different for different installations. The line # in the 'gnome' target file is [L13](https://github.com/dnschneid/crouton/blob/master/targets/gnome#L13).

When the crouton install fails, just use `enter-chroot` and answer '`n`' when asked if you want to let it finish.
You'll then drop to a shell and you can edit `/prepare.sh` replacing `gnome-session-fallback` with `gnome-session-flashback`.

You can exit then and re-enter the chroot using `enter-chroot -u root` to let it finish the setup and add the user, etc.

Hope this helps,
-**DennisL**

**Comment by smibarber** [comment]
Fixed via #2818
